frontend/packages/console-dynamic-plugin-sdk/src/coderefs/coderef-resolver.ts

```diff
@@ -12,7 +12,12 @@ import {
 
 // TODO(vojtech): support code refs at any level within the properties object
 
-export const codeRefSymbol = Symbol('CodeRef');
+const codeRefSymbol = Symbol('CodeRef');
+
+export const applyCodeRefSymbol = <T = any>(ref: CodeRef<T>) => {
```

Comment:
nit: would be nice to specify return type explicitly
```suggestion
export const applyCodeRefSymbol = <T = any>(ref: CodeRef<T>): CodeRef<T> => {
```

Reply:
@rawagner OK, I can do that in a follow-up PR.

TypeScript infers the return type to `CodeRef<T>` anyway, due to `return ref;` being the single exit point of the function.

For simple functions (e.g. low cyclomatic complexity) I prefer to omit return type declaration and rely on type inference.